tests/integration/data/regression_output/linux/slabs | filter obj.name == "kmalloc-8" | member cpu_slab | percpu 0 2 1

```diff
@@ -0,0 +1,2 @@
+(struct kmem_cache_cpu *)0xffffa0897fc27040
+sdb: percpu: available CPUs [0-1] - requested CPU 2
```

Comment:
How is regression testing performed? Do we know for sure that the test environment will have 2 CPUs?

Reply:
This is ran against a reference crash dump that we have and we know that it will always have 2 CPUs on it.

Reply:
Ah okay, thanks for the info